python/bind_data_access.h

```diff
@@ -225,20 +247,22 @@ template <class... Ts> struct as_VariableViewImpl {
           return py::cast(data[0], py::return_value_policy::reference_internal,
                           obj);
         },
-        get<get_values>(view));
+        *get<get_values>(view));
   }
   // Return a scalar variance from a variable, implicitly requiring that the
   // variable is 0-dimensional and thus has only a single item.
   template <class Var> static py::object variance(py::object &obj) {
     auto &view = obj.cast<Var &>();
+    if (!view.hasVariances())
+      return py::none();
```

Comment:
Shouldn't you do the same in `value`?

Reply:
@SimonHeybrock, you've said where is no way to have a variable without values. Am I wrong?

Reply:
This code is also used with `Var = DataConstProxy`, which may have no values.

Reply:
Yes, and hasData() is checked in the proper place. I simply can't find the case where I need a check `if (view)`. Can you?

Reply:
Have you tried: `dataset['a'].values` with a coord-only data item?